src/screens/Ticketing/Purchase/Overview/components/ProductSelectionByProducts.tsx

```diff
@@ -54,9 +54,19 @@ export default function ProductSelectionByProducts({
           keyExtractor={(u) => u.id}
           itemToText={(fp) => getReferenceDataName(fp, language)}
           hideSubtext={hideTravellerDescriptions}
-          itemToSubtext={(fp) =>
-            getTextForLanguage(fp.description ?? [], language) ?? 'Unknow'
-          }
+          itemToSubtext={(fp) => {
+            const descriptionMessage =
+              getTextForLanguage(fp.description ?? [], language) ?? 'Unknow';
+            if (fp.warningMessage) {
+              const warningMessage = getTextForLanguage(
+                fp.warningMessage,
+                language,
+              );
+              return `${descriptionMessage}\n${warningMessage}`;
+            }
+
+            return descriptionMessage;
+          }}
```

Comment:
No `description` on the `PreassignedFareProduct` shouldn't fallback to subtext "Unknown", but instead just not show a subtext. Also by the domain model it is possible for a `PreassignedFareProduct` to have a `warningMessage`, but no `description`, so the code should support that as well.